sycl/test/basic_tests/queue/queue_parallel_for_interface.cpp

```diff
@@ -0,0 +1,81 @@
+// UNSUPPORTED: cuda
+// CUDA does not support unnamed lambdas.
+//
+// RUN: %clangxx -fsycl -fsyntax-only -fsycl-unnamed-lambda %s -o %t.out
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsyntax-only -fsycl-unnamed-lambda %s -c -o %t.out
```
I guess linking can be omitted.

Reply:
It tells me that -c is unknown option. Anyway, we launch this test with `syntax-only` flag that doesn't spend run-time